plugins/google_drive/komand_google_drive/actions/copy_file/action.py

```diff
@@ -0,0 +1,30 @@
+import insightconnect_plugin_runtime
+from .schema import CopyFileInput, CopyFileOutput, Input, Output, Component
+
+# Custom imports below
+from insightconnect_plugin_runtime.exceptions import PluginException
+from googleapiclient.errors import HttpError
+
+
+class CopyFile(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="copy_file", description=Component.DESCRIPTION, input=CopyFileInput(), output=CopyFileOutput()
+        )
+
+    def run(self, params={}):
+        file_id = params.get(Input.FILE_ID)
+        folder_id = params.get(Input.FOLDER_ID)
+        new_file_name=params.get(Input.NEW_FILE_NAME)
+        try:
+
+            if new_file_name:
+                metadata_body = {"name": new_file_name,"parents": [folder_id]}
+            else:
+                metadata_body = {"parents": [folder_id]}
+
```

Comment:
You can use clean method from `insightconnect-plugin-runtime.helpers`
```suggestion
            metadata_body = clean({"name": new_file_name, "parents": [folder_id]})
```